src/templates/_common/components/forms.jsx

```diff
@@ -44,14 +44,38 @@ export const FormRow = ({
                         const option_id = `${id}_option_${i}`;
                         return (
                             <div key={value}>
-                                <input type='radio' name={id} id={option_id} value={value} defaultChecked={value === default_option} />
-                                <label htmlFor={option_id}><span data-balloon={data_balloon_text} data-balloon-length='medium'>{label_text}</span></label>
+                                <input
+                                    type='radio'
+                                    name={id}
+                                    id={option_id}
+                                    value={value}
+                                    defaultChecked={value === default_option}
+                                />
+                                <label htmlFor={option_id}>
+                                    <span
+                                        data-balloon={data_balloon_text}
+                                        data-balloon-length='medium'
+                                    >{label_text}
+                                    </span>
+                                </label>
                             </div>
                         );
                     })}
                 </div>
             );
         }
+        if (type === 'textarea') {
+            return (
+                <React.Fragment>
+                    <textarea
+                        className={className}
+                        id={id}
+                        {...attributes}
```

Comment:
same here, as far as I remember our convention is to break when there are four or more attributes on the same line, here there are only three.